apps/dot_ext/loggers.py

```diff
@@ -117,8 +122,19 @@ def set_session_auth_flow_trace(request, auth_flow_dict):
     '''
     Set auth flow related items in the session from a dictionary.
     '''
-    for k in SESSION_AUTH_FLOW_TRACE_KEYS:
-        request.session[k] = auth_flow_dict.get(k, None)
+    if request.session:
+        for k in SESSION_AUTH_FLOW_TRACE_KEYS:
+            if k in auth_flow_dict:
+                request.session[k] = auth_flow_dict.get(k, None)
```

Comment:
NOTE: Similar here. Only set a value in the `request.session`,  if it has a key in the `auth_flow_dict` dictionary.

Reply:
Since you are checking for the presence of the key, you don't need the None default value correct?

Reply:
> Since you are checking for the presence of the key, you don't need the None default value correct?

@njdister Good catch! I changed similar code related to that after adding that statement above, but didn't catch this one. TY!